plate_mapper/plate_mapper.py

```diff
@@ -68,22 +96,62 @@ def plate_mapper(input_f, barseq_f, output_f):
     barseq_f.close()
     print('  Done.')
     # Write output sequencing run file
+    samples = {}
```

Comment:
Thanks for letting me know this trick! So with this solution I can do:
```
from collections import Counter
samples = []
for ...
    samples.append(sample)
samples = Counter(samples)
```
Is that right? How is it advantageous over the original way? (I guess that is performance?)